Sources/ProjectDescription/Template.swift

```diff
@@ -0,0 +1,76 @@
+public struct Template: Codable {
```

Comment:
Since this is part of the public interface, I'd document the attributes and the methods. That way the user can get inline documentation while editing the template on Xcode.